sputnikvm/src/vm/transaction.rs

```diff
@@ -77,22 +77,27 @@ impl Transaction {
             Transaction::MessageCall {
                 address, caller, gas_price, gas_limit, value, data
             } => {
+                account_state.require(caller)?;
+                account_state.require_code(address)?;
+
                 if !is_code {
-                    let nonce = account_state.nonce(caller)?;
+                    let nonce = account_state.nonce(caller).unwrap();
                     account_state.set_nonce(caller, nonce + M256::from(1u64)).unwrap();
                 }
 
                 Ok(Context {
                     address, caller, data, gas_price, value,
                     gas_limit: gas_limit - upfront,
-                    code: account_state.code(address)?.into(),
+                    code: account_state.code(address).unwrap().into(),
```

Comment:
why not stick to `?` macro?
my rule of thumb, is to check whether it is can be unwrapped with `if` statement, and only then unwrap